### PR TITLE
VU#852201 Layer-2 vulnerabilities

### PR DESCRIPTION
**Pull request by sei-vsarvepalli**
These vulnerabilities were reported by Etienne Champetier @champtar https://github.com/champtar [champetier.etienne@gmail.com](mailto:champetier.etienne@gmail.com) via CERT/CC to multiple vendors.

### CODE REVIEW

File: 2021/27xxx/CVE-2021-27853.json [hunk 1]
```diff
@@ -4,15 +4,121 @@
     "data_version": "4.0",
```
Comment (cve-team) [inline 2021/27xxx/CVE-2021-27853.json:4]: Invalid source.

File: 2021/27xxx/CVE-2021-27854.json [hunk 2]
```diff
@@ -4,15 +4,124 @@
     "data_version": "4.0",
```
Comment (cve-team) [inline 2021/27xxx/CVE-2021-27854.json:4]: Invalid source.

File: 2021/27xxx/CVE-2021-27861.json [hunk 3]
```diff
@@ -4,15 +4,132 @@
     "data_version": "4.0",
```
Comment (cve-team) [inline 2021/27xxx/CVE-2021-27861.json:4]: Invalid source.

File: 2021/27xxx/CVE-2021-27862.json [hunk 4]
```diff
@@ -4,15 +4,107 @@
     "data_version": "4.0",
```
Comment (cve-team) [inline 2021/27xxx/CVE-2021-27862.json:4]: Invalid source.

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2021/27xxx/CVE-2021-27853.json (hunk 1 above)
Comment (sei-vsarvepalli) [reply 2021/27xxx/CVE-2021-27853.json:4]: I believe this is `references/reference_data/*/refsource`  issue? Should be fixed if that is the case.

**Comment by sei-vsarvepalli** [comment]
Seems like multiple errors in JSON apart from source. Closing this PR and starting a new one.

**Pull request closed without merging**